src/main/java/com/wishlist/enums/Categories.java

```diff
@@ -0,0 +1,35 @@
+package com.wishlist.enums;
+    public enum Categories {
+
+        GAMES("games"),
+        CLOTHING("clothing"),
+        ACCESSORIES("accessories"),
+        TOYS("toys"),
+        HEALTH("health"),
+        BEAUTY("beauty"),
+        HOME("home"),
+        BOOKS("books"),
+        MOVIES("movies"),
+        MUSIC("music"),
+        SPORTS("sports"),
+        FURNITURE("furniture"),
+        ELECTRONICS("electronics"),
+        PETS("pets");
+
+        private  final String category;
+
+        Categories(final String category){this.category = category;}
```

Comment:
Not formatted before commit. I strongly advise using IntelliJ, which does such postprocessing automatically for you before committing.